src/models/thread.ts

```diff
@@ -90,7 +90,39 @@ export class Thread extends ReadReceipt<ThreadEmittedEvents, ThreadEventHandlerM
 
     private reEmitter: TypedReEmitter<ThreadEmittedEvents, ThreadEventHandlerMap>;
 
+    /**
+     * The last event in this thread, if we don't yet have this in the timeline.
+     *
+     * When we run processRootEvent (which I think happens during the setting-up
+     * of the thread), we set this to the event pointed to by the server in
+     * latest_event https://github.com/matrix-org/matrix-spec-proposals/blob/main/proposals/3440-threading-via-relations.md#event-format
+     * that came through with the thread root.
```

Comment:
probably best to refer to the stable spec rather than an MSC:

```suggestion
     * `latest_event` [1] that came through with the thread root.
     *
     * [1]: https://spec.matrix.org/v1.8/client-server-api/#server-side-aggregation-of-mthread-relationships
```